src/app/application.cpp

```diff
@@ -521,7 +521,7 @@ void Application::processParams(const QStringList &params)
 
         if (param.startsWith(QLatin1String("@addPaused=")))
         {
-            torrentParams.addPaused = param.midRef(11).toInt() ? TriStateBool::True : TriStateBool::False;
+            torrentParams.addPaused = param.midRef(11).toInt() ? true : false;
```

Comment:
maybe `torrentParams.addPaused = (param.midRef(11).toInt() != 0);`

Reply:
Ok. I was too lazy to do it right away, and went to the nearest path.